espanol/granhermano/granhermano.md

```diff
@@ -0,0 +1,5 @@
+La guerra es la paz. La libertad es la esclavitud. La ignorancia es la fuerza 
```

Comment:
Hi Casey

I have made the fixes in my **local** master, should I push my master now in order to update the main one?  "_git push origin master_"

Or do I need to create another pull request?

Thanks

Christopher